Documentation/FrameworkOverview.md

```diff
@@ -8,196 +8,219 @@ learning about new modules and finding more specific documentation.
-non-RAC code into the world of signals.
+The method returns a [Signal](#signals) and an [Observer](#observers). 
+This signal can be controlled by sending events to the observer. This 
+can be extremely useful for bridging non-RAC code into the world of signals.
 
 For example, instead of handling application logic in block callbacks, the
-blocks can simply send events to a shared subject instead. The subject can then
-be returned as a [RACSignal][], hiding the implementation detail of the
-callbacks.
+blocks can simply send events to a shared observer instead. The signal 
+can be returned, hiding the implementation detail of the callbacks.
 
-Some subjects offer additional behaviors as well. In particular,
-[RACReplaySubject][] can be used to buffer events for future
-[subscribers](#subscription), like when a network request finishes before
-anything is ready to handle the result.
+## Signal Producers
 
-### Commands
+A **signal producer**, represented by the [SignalProducer][] class, creates 
+Signals and performs side effects.
 
-A **command**, represented by the [RACCommand][] class, creates and subscribes
-to a signal in response to some action. This makes it easy to perform
-side-effecting work as the user interacts with the app.
+They can be used to represent operations or tasks, like network 
+requests, where each invocation of `start()` will create a new underlying 
+operation. The produced signal is returned to the caller, who can observe
+the result of the task by observing the signal.
 
-Usually the action triggering a command is UI-driven, like when a button is
-clicked. Commands can also be automatically disabled based on a signal, and this
-disabled state can be represented in a UI by disabling any controls associated
-with the command.
+Because of the behavior of `start()`, different Signals created from the 
+producer may see a different version of events, the events may arrive in a 
+different order between signals, or the stream might be completely different!
+However, this behavior ensures that consumers will receive the results, 
+in contrast to a plain signal that might send results befor any observers 
+are attached.
 
-On OS X, RAC adds a `rac_command` property to
-[NSButton][NSButton+RACCommandSupport] for setting up these behaviors
-automatically.
+Starting a signal returns a [disposable](#disposables) which can be used to 
+interrupt/cancel the work associated Signal.
 
-### Connections
+Just like Signals, Signal Producers can also be manipulated. Every signal 
+operator can be `lift`ed to operate upon Signal Producers instead. In other
+words, `lift` creates a new Signal Producer which will apply the given 
+signal operator to __every__ signal created from the producer. 
+<!-- TODO: |> is a different kind of operator as signal operators. 
+How can this be clearified? -->
+Again, the `|>` operator can be used to apply signal operators to a signal
+producer which implicitly lifts the operator.
```

Comment:
Maybe another sentence that there are additional functions for signal producer transformations for controlling _how_ and _when_ the work is started.